2024_wsb/Inc/multiSensorADC.h

```diff
@@ -3,9 +3,16 @@
 
 #include "stm32f4xx_hal.h"
 
-#define NUM_ADC_CHANNELS 3  // BrakeIr, Sensor2, Sensor3
+#define NUM_ADC_CHANNELS 3 // BrakeIr, Sensor2, Sensor3
 
+// Each element of this array will hold the value of the corresponding ADC channel
+// TODO: Acc the order might not be as below, need to test
+// buf[0] = BrakeIr, buf[1] = Sensor2, buf[2] = Sensor3, ...
 extern uint32_t multiSensorADCBuf[NUM_ADC_CHANNELS];
```

Comment:
I am not sure if this is intended, but why is this array included here? This is not being used right

Reply:
It is being used by the dma. But if other code needs access to the underlying dma buffer, they can access it from this header file